doc/releases/changelog-0.39.0.md

```diff
@@ -250,65 +411,68 @@
   [(#6288)](https://github.com/PennyLaneAI/pennylane/pull/6288)
 
 * The number of diagonalizing gates returned in `qml.specs` now follows the `level` keyword argument 
-  regarding whether the diagonalizing gates are modified by device, instead of always counting 
-  unprocessed diagonalizing gates.
+  regarding whether the diagonalizing gates are modified by device, instead of always counting unprocessed 
+  diagonalizing gates.
   [(#6290)](https://github.com/PennyLaneAI/pennylane/pull/6290)
 
-* A more sensible error message is raised from a `RecursionError` encountered when accessing properties and methods of a nested `CompositeOp` or `SProd`.
+* A more sensible error message is raised from a `RecursionError` encountered when accessing properties 
+  and methods of a nested `CompositeOp` or `SProd`.
   [(#6375)](https://github.com/PennyLaneAI/pennylane/pull/6375)
 
-* Moved the calculation of `shift_len = len(shifts)` outside of a loop in the `QFT.compute_decomposition` static method, reducing redundant recalculations and improving performance.
+* The performance of the decomposition of `qml.QFT` has been improved.
   [(#6434)](https://github.com/PennyLaneAI/pennylane/pull/6434)
 
 <h3>Breaking changes 💔</h3>
 
-* `AllWires` validation in `QNode.construct` has been removed. 
+* The `AllWires` validation in `QNode.construct` has been removed. 
   [(#6373)](https://github.com/PennyLaneAI/pennylane/pull/6373)
```

Comment:
This is another case of an entry that needs to be rewitten to make it clear what the change is from a user perspective. What will now be allowed that wasn't before? Is there a code example to illustrate this?

How is this a break in behaviour?